### PR TITLE
small agenda improvments

### PR DESCRIPTION
**Pull request by ostcar**
I need this merged into master before I can fix the tests for #1444

### CODE REVIEW

File: openslides/agenda/models.py [hunk 1]
```diff
@@ -209,12 +203,17 @@ def weight_form(self):
...
+        well. If with_children is False, all children will be children of the
+        parent of the item.
         """
         if not with_children:
             for child in self.get_children():
                 child.move_to(self.parent)
                 child.save()
-        super(Item, self).delete()
+        super().delete()
+        # TODO: Try to delete the rebuild call
```
Comment (normanjaeckel) [inline openslides/agenda/models.py]: Proposal: ~~delete~~ remove

**Comment by normanjaeckel** [comment]
Can you please add a short description of the improvements or changes in the commit message (starting with a capital letter just for prettiness)?

File: openslides/agenda/models.py (hunk 1 above)
Comment (ostcar) [inline openslides/agenda/models.py]: done

**Comment by ostcar** [comment]
All done.

The problem was not the agenda code, but the old agenda tests. I changed small parts of the code by rewriting the tests.

**Comment by normanjaeckel** [comment]
Rebase and merge.

**Pull request merged**